src/libstd/keyword_docs.rs

```diff
@@ -77,3 +77,54 @@ mod let_keyword { }
 ///
 /// [book]: https://doc.rust-lang.org/book/second-edition/ch05-01-defining-structs.html
 mod struct_keyword { }
+
+#[doc(keyword = "enum")]
+//
+/// The `enum` keyword.
+///
+/// The `enum` keyword is used to define an enum type.
+///
+/// Example:
+///
+/// ```
+/// enum Foo {
+///     Value1,
+///     Value2,
+///     Value3(u32),
+///     Value4 { x: u32, y: u64 },
+/// }
+/// ```
+///
+/// Very convenient to handle different kind of data. To get what value an enum instance is,
```

Comment:
I'd use the word "variant" instead of "value"; and "see" instead of "get.
So: "To see which variant a value of an enum type is of, you can use pattern matching on the value:"